salt/modules/junos.py

```diff
@@ -72,6 +71,81 @@
 __proxyenabled__ = ["junos"]
 
 
+class HandleFileCopy:
```

Comment:
could this class be placed after the __virtual__, since that is usually kept to the top of a file

Reply:
will do this change

Reply:
Fixed via https://github.com/saltstack/salt/pull/57399/commits/f308a577738f4e70d27927594cf3c26bab79b4b8